docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -72,6 +71,134 @@ FIELDS TERMINATED BY '|'
 LOCATION 'alluxio://master_hostname:port/ml-100k';
 ```
 
+## Create Internal Table from Files Located in Alluxio
+
+Hive can create internal tables from files stored on Alluxio and Hive will manage the lifecycle of internal tables.
+When you drop an internal table, Hive drops the table metadata and data both.
```

Comment:
Yes , @apc999 I have tested against this situation and Hive delete files in alluxio://master_hostname:port/ml-100k